### PR TITLE
Travis CI Setup

### PR DESCRIPTION
**Pull request by jmartin84**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        ],
+        "semi": [
+            2,
+            "always"
+        ]
+    },
+    "env": {
+        "es6": true,
+        "node": true,
+        "browser": true
```
Comment (CWSpear) [inline .eslintrc:23]: Should add `mocha`.

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    },
+    "extends": "eslint:recommended"
+}
```
Comment (CWSpear) [inline .eslintrc:26]: Do we care to be consistent with newlines at the end of files? `.editorconfig`?

Comment (jmartin84) [inline .eslintrc:26]: yeah, I was gonna create an issue for you to set us up a .editorconfig.

that being said this file is generated by eslint, I do think we will have rule changes to make as we go - this is only a baseline.

Comment (CWSpear) [inline .eslintrc:26]: ah, ok

**Pull request merged**